packages/web/README.md

```diff
@@ -60,6 +62,11 @@ type Config = {
   debug: bool; // enable or disable useful console.log outputs
   proxyToWorker: bool; // Whether to proxy the calculations to a web worker. (Default true)
   model: 'small' | 'medium'; // The model to use. (Default "medium")
+  output: {
+    type: 'image/png' | 'image/jpeg' | 'image/webp'; // The output type. (Default "image/png")
```

Comment:
Is this correct? It seems like that should 
```suggestion
    format: 'image/png' | 'image/jpeg' | 'image/webp'; // The output type. (Default "image/png")
```

Reply:
Good point. There was another error